pkg/external_repos/introspect.go

```diff
@@ -124,6 +141,27 @@ func IntrospectAll() (int64, []error) {
 	return total, errors
 }
 
+func needsIntrospect(repo *dao.Repository) (bool, string) {
+	if repo == nil {
+		return false, "Cannot introspect nil Repository"
+	}
+
+	if repo.Status != config.StatusValid {
+		return true, fmt.Sprintf("The Status field content differs from '%s' for Repository.UUID = %s", config.StatusValid, repo.UUID)
```

Comment:
```suggestion
		return true, fmt.Sprintf("Introspection started: the Status field content differs from '%s' for Repository.UUID = %s", config.StatusValid, repo.UUID)
```
Add "Introspection started" to the front, just so it is clear that introspection was run